sonar-ruby-plugin/src/main/java/org/sonarsource/ruby/converter/RubyVisitor.java

```diff
@@ -49,6 +59,26 @@
 
   private final TreeMetaDataProvider metaDataProvider;
 
+  private static final Map<String, Operator> BINARY_OPERATOR_MAP = Collections.unmodifiableMap(Stream.of(
```

Comment:
This is bit unusual way to build a map, I suggest we either add dependency on Guava and use ImmutableMap.of , or use HashMap put . I think adding guava is fine